includes/admin/tools.php

```diff
@@ -1377,7 +1377,8 @@ function edd_tools_sysinfo_get() {
 			continue;
 
 		$update = ( array_key_exists( $plugin_path, $updates ) ) ? ' (needs update - ' . $updates[$plugin_path]->update->new_version . ')' : '';
-		$return .= $plugin['Name'] . ': ' . $plugin['Version'] . $update . "\n";
+		$plugin_url = isset( $plugin['PluginURI'] ) && ! empty( $plugin['PluginURI'] ) ? $plugin['PluginURI'] : $plugin['AuthorURI'];
+		$return .= $plugin['Name'] . ': ' . $plugin['Version'] . $update . ' - ' . $plugin_url . "\n";
```

Comment:
Just a question for you--as you look at the list, if a plugin needs an update, is it more helpful to have them in the order you do already, or would you want the `$update` bit at the end? I'm seeing this:

```
Advanced Custom Fields PRO: 5.9.5 (needs update - 5.9.9) - https://www.advancedcustomfields.com
AffiliateWP: 2.7-beta1 (needs update - 2.8-alpha2) - https://affiliatewp.com
```
(I am just asking--either way is fine with me.)

Reply:
I think its easier and more "in front" of you to see the Needs update message right at the start than having to perhaps scroll to the right if its a long URI

Reply:
@mihaijoldis this is looking great, but I discovered a plugin in my site which meets none of these parameters, so the output is:

```
Plugin Name: 1.5 - 
```

What would you think of updating the logic a little to drop that if the `$plugin_url` can't be retrieved? Something like:

```php
		if ( $plugin_url ) {
			$plugin_url = ' &mdash; ' . $plugin_url;
		}
		$return .= $plugin['Name'] . ': ' . $plugin['Version'] . $update . $plugin_url . "\n";
```
(I threw in the `&mdash;` as our extension names have a hyphen in them already and thought that might help differentiate a bit.)